kahuna/public/stylesheets/main.css

```diff
@@ -406,7 +406,7 @@ textarea.ng-invalid {
     left: 0;
     right: 0;
     top: 5px;
-    z-index: 2;
+    z-index: 10;
```

Comment:
is it worth using a similar strategy/style-guide as frontend regarding z-index values? (unfortunately, can't find the link).